src/shapleypy/game.py

```diff
@@ -0,0 +1,83 @@
+from typing import Iterable, List, Tuple
+
+import numpy as np
+
+from shapleypy.coalition import Coalition
+from shapleypy.protocols import Player, Value
+
+Players = Iterable[Player]
+Coalitions = Iterable[Coalition]
+
+
+class Game:
+    def __init__(self, number_of_players: int) -> None:
+        self.number_of_players: int = number_of_players
+        self._values: np.ndarray = np.zeros(2**number_of_players, dtype=Value)
+        self._init_values()
+
+    def set_value(self, coalition: Coalition | Players, value: Value) -> None:
```

Comment:
Přemýšlím, že by i bylo šikovné zadávat koalici i pomocí její binární reprezentace.

Reply:
I opened an issue #9. We should discuss it more.